src/VecSim/algorithms/brute_force/brute_force.h

```diff
@@ -11,19 +11,18 @@
 
 using spaces::dist_func_t;
 
-class BruteForceIndex : public VecSimIndex {
+class BruteForceIndex : public VecSimIndexAbstract {
```

Comment:
For readability, I think we need function docs here now that we have inheritance, 
for example, we have `addVector` from the interface but also `appendVector` to be used in it